Makefile

```diff
@@ -152,10 +152,12 @@ fast-bench:
 bench:
 	time -p sh -c "go test -run _NONE_ -bench . -benchmem -benchtime 7s -cpu 2 ./test/... | grep -v '^ok ' | grep -v '\[no test files\]' | grep -v '^PASS'"
 
+.PHONY: $(PROGS)
```

Comment:
I marked them as `.PHONY` with the same reason as the `$(EXAMPLE_SERVICES)` target, should they not be?